UPEI-Android/group-project-mindjogg
# Dev frontend documents

### Summary of PR 
Adds Technical documents for frontend 


### Source
Closes 
#208 to #215 (all of them)


### Review and Testing Steps

1.  Pull this branch
2. Check all the documents
3. Make sure everything make sense and understandable
4. Give your feedback


### Note
